### PR TITLE
Fix: Change the `sliderMax` based on the Image's width and height value

### PR DESCRIPTION
**Pull request by blankart**

### CODE REVIEW

**Review by bfintal — CHANGES_REQUESTED** [review]
The max should change depending on the width or height, whichever is higher

**Comment by blankart** [comment]
> The max should change depending on the width or height, whichever is higher

There are units in both height and width. Border radius is strictly px

**Comment by bfintal** [comment]
> There are units in both height and width. Border radius is strictly px

We can use the max value of the width and height whichever is in `px`. Then is both are not `px`, then use `100` instead.

File: src/block-components/image/edit.js [hunk 1]
```diff
@@ -41,6 +42,25 @@ const Controls = props => {
...
+
+		if ( attributes.imageWidthUnit === 'px' ) {
+			return attributes.imageWidth
+		}
+
+		if ( [ '', 'px' ].includes( attributes.imageHeightUnit ) ) {
+			return ( ( attributes.imageHeight || 100 ) / 2 )
+		}
+
+		return 50
```
Comment (bfintal) [inline src/block-components/image/edit.js:61]:
```suggestion
		return 100
```

File: src/block-components/image/edit.js [hunk 2]
```diff
@@ -41,6 +42,25 @@ const Controls = props => {
 		return { imageData: { ...image } }
 	}, [ attributes.imageId ] )
 
+	const borderRadiusSliderMax = useMemo( () => {
+		if (
+			attributes.imageWidthUnit === 'px' &&
```
Comment (bfintal) [inline src/block-components/image/edit.js:47]:
```suggestion
			[ '', 'px' ].includes( attributes.imageWidthUnit ) &&
```

File: src/block-components/image/edit.js [hunk 3]
```diff
@@ -41,6 +42,25 @@ const Controls = props => {
 		return { imageData: { ...image } }
 	}, [ attributes.imageId ] )
 
+	const borderRadiusSliderMax = useMemo( () => {
+		if (
+			[ '', 'px' ].includes( attributes.imageWidthUnit ) &&
+			( [ '', 'px' ].includes( attributes.imageHeightUnit ) )
```
Comment (bfintal) [inline src/block-components/image/edit.js:48]:
```suggestion
			[ '', 'px' ].includes( attributes.imageHeightUnit )
```

File: src/block-components/image/edit.js [hunk 4]
```diff
@@ -41,6 +42,25 @@ const Controls = props => {
 		return { imageData: { ...image } }
 	}, [ attributes.imageId ] )
 
+	const borderRadiusSliderMax = useMemo( () => {
+		if (
+			[ '', 'px' ].includes( attributes.imageWidthUnit ) &&
```
Comment (bfintal) [inline src/block-components/image/edit.js:47]:
```suggestion
			attributes.imageWidthUnit === 'px' &&
```

**Pull request merged**